fe/fe-core/src/main/java/com/starrocks/load/loadv2/LoadingTaskPlanner.java

```diff
@@ -257,18 +270,25 @@ public void buildShufflePlan(TUniqueId loadId, List<List<TBrokerFileStatus>> fil
 
         List<Long> partitionIds = getAllPartitionIds();
         // Parallel pipeline loads are currently not supported, so disable the pipeline engine when users need parallel load
-        OlapTableSink olapTableSink = new OlapTableSink(table, tupleDesc, partitionIds, parallelInstanceNum <= 1, writeQuorum);
+        OlapTableSink olapTableSink = new OlapTableSink(table, tupleDesc, partitionIds, true, writeQuorum);
         olapTableSink.init(loadId, txnId, dbId, timeoutS);
         olapTableSink.complete();
 
         // 6. Sink plan fragment
         sinkFragment.setSink(olapTableSink);
-        // At present, we only support dop=1 for olap table sink.
-        // because tablet writing needs to know the number of senders in advance
-        // and guaranteed order of data writing
-        // It can be parallel only in some scenes, for easy use 1 dop now.
-        sinkFragment.setPipelineDop(1);
-        sinkFragment.setParallelExecNum(parallelInstanceNum);
+        // For shuffle broker load, we only support tablet sink dop = 1
+        // because for tablet sink dop > 1, local passthourgh exchange will influence the order of sending,
+        // which may lead to inconsisten replica for primary key.
```

Comment:
Also add the unique key

Reply:
because we have implemented single replica load, shuffle service will be disabled lator, no need to add adjustment for tablet sink dop for shuffle service